src/gfx/qt.cpp

```diff
@@ -147,10 +151,12 @@ int GfxProcQT::getExifOrientation(QString &filePath)
             default:
                 // Skip any other sections.
                 delete data;
+                data = NULL;
                 break;
         }
     }
 
+    delete data;
```

Comment:
Shouldn't we move this `delete data` to the end of `case M_EXIF:`?

Reply:
In that case, there would be a double `delete` because that case finishes on the `default` case. Anyway, I will improve the code to make it more readable.